components/list/test/list-item-drag-mixin.test.js

```diff
@@ -106,6 +106,9 @@ describe('ListItemDragDropMixin', () => {
 
 		it('should have the list item go back to normal when dragging ends', async() => {
 			const dragArea = element.shadowRoot.querySelector('.d2l-list-item-drag-area');
+			dispatchDragEvent(dragArea, 'dragstart');
+			await oneEvent(dragArea, 'dragstart');
+			await element.updateComplete;
```

Comment:
This was causing a JS error but wasn't failing the test. The start event needs to fire first before the end event.